debian/control

```diff
@@ -40,6 +40,7 @@ Depends: ${python3:Depends}, ${misc:Depends}, ${extra:Depends},
          ${shlibs:Depends},
          landscape-common (= ${binary:Version}),
          python3-pycurl,
+         python3-netifaces,
```

Comment:
landscape-sysinfo is actually part of landscape-common (shipped with ubuntu-server). That dependency should be in the other Package. (cloud-init pulls it so it likely will be already there, but still)

Reply:
Fixed, thanks!